docs/oscar_references.bib

```diff
@@ -519,3 +519,19 @@ @Book{Zie95
   doi           = {10.1007/978-1-4613-8431-1},
   url           = {https://doi.org/10.1007/978-1-4613-8431-1}
 }
+
+@Book{CLS11,
+  Author = {David A. {Cox} and John B. {Little} and Henry K. {Schenck}},
+  Title = {{Toric varieties}},
+  FJournal = {{Graduate Studies in Mathematics}},
+  Journal = {{Grad. Stud. Math.}},
+  ISSN = {1065-7338},
+  Volume = {124},
+  ISBN = {978-0-8218-4819-7},
+  Pages = {xxiv + 841},
+  Year = {2011},
+  Publisher = {Providence, RI: American Mathematical Society (AMS)},
+  Language = {English},
+  MSC2010 = {14-01 14M25 14C17 14C15},
+  Zbl = {1223.14001}
+}
```

Comment:
If it is not too much trouble, could you apply the instructions in `docs/README.md` for updating the bib?

Reply:
Sorry, yes, I was not aware of this.